pkg/infrastructure/services/deployer.go

```diff
@@ -344,5 +346,11 @@ func (s *serviceDeployer) configureMonitoring() (time.Duration, error) {
 	} else if failedVerifyAddon {
 		return reconciliationPeriodMonitoringEndpointNotAvailable, nil
 	}
-	return 0, nil
+
+	reconcileAfter, err := configureGrafanaDashboards(s.client, s.instance, s.scheme, s.getNamespace())
+	if err != nil {
+		return reconciliationPeriodAfterDashboardsError, err
```

Comment:
In case reconciliation returns error then reconciliation period is ignored and default error behaviour is used (reconciling every second for first few reconciliation loops, then exponentially raising interval).

Reply:
Hi @sutaakar , thanks for the catch! I've modified the function so to return 
```
reconciliationPeriodAfterDashboardsError, nil
```
in case of error.

Reply:
But please, do not ignore the error. Log it as a WARN at least.

Reply:
Done :+1: